packages/compass-aggregations/src/components/stage-preview/stage-preview.jsx

```diff
@@ -92,19 +92,26 @@ class StagePreview extends Component {
       }
       return (<div className={styles['stage-preview-out']} />);
     }
+
+    const previewOutText =
+      process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR !== 'true'
```

Comment:
Should we 
```suggestion
      process?.env?.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR !== 'true'
```
For cloud compatibility?

Reply:
webpack always polyfills `process.env` (and replaces usage of process.env.<var_name> with a static value if provided) so there is no need to do this

Reply:
Just checked the codebase and the way we access these with `global?.process?.` in a few places is problematic, webpack provide plugin doesn't pick those up, thanks for highlighting this